hudi-client/src/main/java/org/apache/hudi/table/HoodieCopyOnWriteTable.java

```diff
@@ -125,6 +128,16 @@ public HoodieWriteMetadata bulkInsertPrepped(JavaSparkContext jsc, String instan
         this, instantTime, preppedRecords, bulkInsertPartitioner).execute();
   }
 
+  @Override
+  public Option<HoodieClusteringPlan> scheduleClustering(JavaSparkContext jsc, String instantTime, Option<Map<String, String>> extraMetadata) {
+    return new ScheduleClusteringActionExecutor(jsc, config, this, instantTime, extraMetadata).execute();
+  }
+
+  @Override
+  public HoodieWriteMetadata clustering(JavaSparkContext jsc, String compactionInstantTime) {
+    return new RunClusteringActionExecutor(jsc, config, this, compactionInstantTime).execute();
+  }
+
```

Comment:
Need to ensure small file handling doesn't toe-step clustering.  In other words, we should employ update rejection policy for small file handling too.

Reply:
Thanks, Will pay attention to this